go/client/commands_non_osx.go

```diff
@@ -17,6 +17,10 @@ func getPlatformSpecificCommands(cl *libcmdline.CommandLine, g *libkb.GlobalCont
 	return []cli.Command{}
 }
 
+func StopLaunchdService(g *libkb.GlobalContext, label string, wait bool) error {
+	return fmt.Errorf("Unsupported on this platform")
```

Comment:
Just repeating myself from above, but I think it would be cleaner if this function weren't _defined_ on non-OSX platforms. That way the compiler would force us to keep platform-specific code out of platform-independent code. On the other hand there's a ton of autofork-specific and brew-specific code all over `main.go`, so maybe I'm just dreaming.